src/basic/meson.build

```diff
@@ -135,6 +135,8 @@ basic_sources = files('''
         smack-util.c
         smack-util.h
         socket-label.c
+        socket-protocol-list.c
+        socket-protocol-list.h
```

Comment:
I don't understand. Why not move the script to src/shared instead?

Reply:
Because I am lazy about updating meson.build :-) Ok, I will move the scripts later.